weldx/schemas/weldx.bam.de/weldx/core/transformations/local_coordinate_system-0.1.0.yaml

```diff
@@ -103,20 +103,18 @@ properties:
     tag: "asdf://weldx.bam.de/weldx/tags/time/timedeltaindex-0.1.*"
   coordinates:
     description: |
-      An ndarray containing the coordinates.
+      A ndarray containing the coordinates.
```

Comment:
I think the previous version was correct due to the way `ndarray` is pronounced. But not 100% sure here